django/contrib/gis/gdal/geomtype.py

```diff
@@ -85,3 +85,11 @@ def django(self):
         elif s == 'Unknown':
             s = 'Geometry'
         return s + 'Field'
+
+    def to_multi(self):
+        """
+        Transform Point, LineString, Polygon, and their 25D equivalents
+        to their Multi... counterpart.
+        """
+        if self.num in (1, 2, 3, 1 + self.wkb25bit, 2 + self.wkb25bit, 3 + self.wkb25bit):
```

Comment:
How about indexing `self._str_types` to get the integers instead of reconstructing them?